docker-entrypoint.d/00-update-uid

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# current www-data UID
+CURRENT_UID=`id -u www-data`
+WANTED_UID=${RUN_AS_UID:-33}
+
+# check if UID need to be updated
+if [ $CURRENT_UID -ne $WANTED_UID ]; then
```

Comment:
@Vampouille that makes sense.. I'm not clear why the ID was not set by default on my instance, but based on what you're saying it doesn't seem like this change needs to be made. Thanks for taking the time to respond!